contributing/code/standards.rst

```diff
@@ -101,33 +109,34 @@ short example containing most features described below::
             );
 
             if (true === $dummy) {
-                return null;
+                return 'something';
             }
 
-            if ('string' === $dummy) {
+            if (is_string($dummy)) {
```

Comment:
Note: I have seen `\is_string(` (backslash-prepended) in the code but not so in the docs

Reply:
This not the same anymore before it did not check if it is a string, but if the string is the same string as `string`....

Reply:
That's intended, as I said:

> [...] and make the logic a bit more realistic (even if still dummy code)

Look at the code inside that `if`, before it was basically either `return substr('string', 0, 5);` or `return ucwords('string');`...